README.md

```diff
@@ -1,67 +1,67 @@
-& LifeCycle Metrics that can then be viewed on the CTO.ai Delivery Metrics
-dashboard.
-
-### Install Agent
+### Install Default Agent
 
 1. From your GitHub repo, click Settings -> Secrets -> New Secret
   - Create CTOAI_TEAM_ID secret using your CTO.ai-issued Team Id.
   - Create CTOAI_EVENTS_API_TOKEN secret using your CTO.ai-issued API Token.
-2. Access the CTO.ai Agent workflow from:
-  - https://github.com/cto-ai/agent/blob/master/.github/workflows/ctoai-agent.yml
-3. Place `ctoai-agent.yml` in the workflows folder of your repo:
-  - .github/workflows/ctoai-agent.yml
-4. You're done! Delivery & LifeCycle Metrics can now be viewed on the Metrics
-   Dashboard.
+2. Copy all of the CTO.ai Agent workflow from:
+  - https://github.com/cto-ai/agent/blob/master/.github/workflows/
+3. Place these workflows in the workflows folder of your repo:
+  - .github/workflows/ or integrate directly with your existing actions.
+4. You're done! Now events from your workflow will be sent!
+
+By default our existing agents assume you use Github Flow. By default we count 
+commits to master as a "successful deployment" and pull requests with status closed 
+as a "failed deployment". However, if you want to get more accurate with measuring your
+actual deployments you should manually instrument your pipeline at this custom stage.
 
 ## Manually Instrument Pipelines using the CTO.ai Action
 
-In addition to the drop-in Agent, you can manually instrument your GitHub
-action workflows with specific LifeCycle events that you specify. For example,
-if you have a GitHub workflow that deploys your application to Azure, you could
-add steps to this workflow that use the CTO.ai Action (cto-ai/action) to store
-metrics. The CTO.ai Action will store the event information that you specify,
-and those metrics can then be visible on the CTO.ai Delivery Metrics Dashboard.
+When instrumenting a custom deployment pipeline, you'll need to isolate the code that you 
+have created for your deployments and add steps for our action to send us the correct info.
 
-The CTO.ai steps that you can add to your workflows look like this:
+The CTO.ai steps that you should add to your workflows look like this:
 
 ```
 - name: Report Build Succeeded
   if: ${{ success() }}
   uses: cto-ai/action@v1-beta
   id: ctoai-build-succeeded
+  env:
+    CTOAI_ACTION_ENVIRONMENT: ${{ secrets.CTOAI_ACTION_ENVIRONMENT }}
+    CTOAI_EVENTS_API_TOKEN: ${{ secrets.CTOAI_EVENTS_API_TOKEN }}
+    CTOAI_TEAM_ID: ${{ secrets.CTOAI_TEAM_ID }}
   with:
-    team_id: ${{ secrets.CTOAI_TEAM_ID }}
-    token: ${{ secrets.CTOAI_EVENTS_API_TOKEN }}
-    stage: "Build"
-    status: "Succeeded"
-    change-id: "your-change-id1"
-    pipeline-id: "your-pipeline-id1"
+    event_name: 'deployment'
+    event_action: 'succeeded'
+    repo: 'org/repo'
+    environment: 'production'
+    branch: 'main'
+    commit: 'cfb11968bb0afe4cad144133996d0da84697cd08'
+    image: '6327219d0db0'
```

Comment:
this is better, but can still be improved. You can get payload info from github `github.ref` for branch on push or `github.pull_request.base.ref` for pull_request.

Commit you could get, image you cannot. Maybe the action can try to "create" an image name but that doesn't make sense.

Reply:
adding our action after deploy events makes more sense, could we just include this and drop the `./github/workflows`?

Reply:
Yup we can add variables into examples once we have static values working with the API.